modules/specialized-guides/pages/large-deployments/overview.adoc

```diff
@@ -21,6 +21,9 @@ There are two main ways to manage large scale deployments.
 You can manage them with a single {productname} Server, or you can use multiple servers in a hub.
 Both methods are described in this book.
 
+With large scale environments one should also consider the usage of {productname} Proxies, for more information see xref:installation-and-upgrade:install-proxy.adoc[].
+Sizing and location of the Proxies will dependent on the deployment topology.
```

Comment:
```suggestion
With large scale environments one should also consider the usage of {productname} Proxies.
Sizing and location of the proxies will dependent on the deployment topology.
For more information about {productname} proxies, see xref:installation-and-upgrade:install-proxy.adoc[].
```
Arguable, but in general we prefer more-info-refs at the end of text chunks.

Thanks for the PR!